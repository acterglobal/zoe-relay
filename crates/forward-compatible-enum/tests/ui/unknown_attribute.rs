use forward_compatible_enum::ForwardCompatibleEnum;

#[derive(ForwardCompatibleEnum)]
#[forward_compatible(unknown_attribute = "value")]  // Unknown attribute!
pub enum UnknownAttribute {
    #[discriminant(1)]
    Variant1,
    Unknown { discriminant: u32, data: Vec<u8> },
}