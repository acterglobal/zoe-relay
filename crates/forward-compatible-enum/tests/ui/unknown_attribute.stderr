error: unknown attribute in forward_compatible
 --> tests/ui/unknown_attribute.rs:4:22
  |
4 | #[forward_compatible(unknown_attribute = "value")]  // Unknown attribute!
  |                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^

error[E0601]: `main` function not found in crate `$CRATE`
 --> tests/ui/unknown_attribute.rs:9:2
  |
9 | }
  |  ^ consider adding a `main` function to `$DIR/tests/ui/unknown_attribute.rs`
